### DISCUSSION TITLE
Other Headers being included in export

### DISCUSSION THREAD

**Discussion by FANNYMAN231** [Budibase/budibase#9773] [Help] [answered]
I'm new to budibase and I'm watching the Budibase University video about adding actions to one of my apps/pages.
I've tried using the template of Employee Portal, and tried adding an export button to the Employee Search screen.
However, when I tried only selecting the First Name, Last Name, and Phone number to be exported only, it includes the header for the other columns as well when exporting it to csv.
In the video, Adding actions, it's showing that only the selected columns in the button/action setting in the exported csv file.
Am I doing something wrong? Is there another setting that I need to do in the table itself or the action button?

**Comment by andz-bb** [comment, marked as answer]
hey @FANNYMAN231 - welcome to Budibase 😊

thanks for letting us know about this, I've checked it out and I'm seeing the same on my end. I've raised it as a bug here: #9776
